package.json

```diff
@@ -61,15 +69,14 @@
     "react-ga": "^2.2.0",
     "react-hot-loader": "next",
     "react-redux": "^5.0.5",
-    "react-router": "^4.1.1",
     "react-router-dom": "^4.1.1",
```

Comment:
`react-router-dom` can probably go too since it's useless without `react-router`